/raid/projects/weustis/conda/envs/weustis/lib/python3.9/site-packages/torch/functional.py:445: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at  /opt/conda/conda-bld/pytorch_1640811803361/work/aten/src/ATen/native/TensorShape.cpp:2157.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
  0%|                                                                                                                                        | 0/4039 [00:04<?, ?it/s]
  0%|                                                                                                                                         | 0/157 [00:00<?, ?it/s]
torch.Size([32, 200]) torch.Size([32])
tensor([[ 0.8305,  0.1118,  1.3762,  ..., -0.7379,  0.9763,  1.1692],
        [ 0.6664, -0.0829,  1.0600,  ..., -0.6813,  1.1214,  0.7908],
        [ 0.4782,  0.3124,  1.1855,  ..., -0.4009,  0.9040,  0.8610],
        ...,
        [ 0.7222,  0.0798,  1.4786,  ..., -0.6832,  1.1770,  1.6038],
        [ 0.5583, -0.0121,  0.9416,  ..., -0.1547,  0.7786,  0.8352],
        [ 1.1342,  0.1426,  1.8726,  ..., -0.9447,  1.0970,  1.9812]],
       device='cuda:0', grad_fn=<MmBackward0>) tensor([14, 90, 15, 61, 26, 29, 16, 20, 78, 47,  8, 65, 42, 37, 73, 94, 63, 37,
        65, 58, 20, 67, 97, 54,  0, 76, 15, 77, 66, 19, 13, 61],
  0%|                                                                                                                                         | 0/157 [00:01<?, ?it/s]
  0%|                                                                                                                                        | 0/4039 [00:02<?, ?it/s]
  0%|                                                                                                                                         | 0/157 [00:00<?, ?it/s]
torch.Size([32, 200]) torch.Size([32])
tensor([[ 0.3517,  0.4278,  0.5460,  ..., -0.2142,  0.7090,  0.5356],
        [ 0.7262,  0.0434,  1.2296,  ..., -0.6474,  1.0702,  1.1274],
        [ 0.4319,  0.1446,  0.8817,  ..., -0.3173,  0.8606,  0.6986],
        ...,
        [ 0.6060,  0.1549,  0.9475,  ..., -0.2767,  0.9582,  0.8644],
        [ 0.7454,  0.0133,  1.1382,  ..., -0.5356,  0.9140,  0.9577],
        [ 0.8958, -0.0691,  1.1700,  ..., -0.5973,  1.0533,  0.9816]],
       device='cuda:0', grad_fn=<MmBackward0>) tensor([74, 41, 78, 76, 33, 44, 73, 73, 69, 26, 79, 42, 46, 45, 63, 60, 62, 82,
        64, 34, 77, 67, 30, 73, 88, 78, 42, 28, 90, 42, 13, 26],
  0%|                                                                                                                                         | 0/157 [00:01<?, ?it/s]
  0%|                                                                                                                                        | 0/4039 [00:00<?, ?it/s]
torch.Size([32, 200]) torch.Size([32])
tensor([[ 0.5140,  0.5543,  1.1209,  ..., -0.6548,  0.9855,  1.0539],
        [ 1.1512,  0.1578,  1.5056,  ..., -1.0654,  1.0871,  1.4307],
        [ 0.9957, -0.0532,  1.2729,  ..., -0.7079,  1.0687,  1.0945],
        ...,
        [ 0.8589, -0.0762,  1.0021,  ..., -0.6812,  0.9624,  0.8267],
        [ 0.9994, -0.0050,  1.1436,  ..., -0.7424,  1.1106,  1.1156],
        [ 0.8616,  0.1343,  0.9189,  ..., -0.7074,  0.8581,  0.6151]],
       device='cuda:0', grad_fn=<MmBackward0>) tensor([33,  3,  7, 61, 71, 34, 95, 47, 98, 32,  1, 37, 12, 74, 79, 27, 48, 35,
        63,  5, 64, 26, 83, 95,  0, 47, 88, 73,  4, 31, 42, 28],
  0%|                                                                                                                                        | 0/4039 [00:01<?, ?it/s]
  0%|                                                                                                                                         | 0/157 [00:01<?, ?it/s]
  0%|                                                                                                                                        | 0/4039 [00:00<?, ?it/s]Exception ignored in: <function _releaseLock at 0x7f085527dca0>
Traceback (most recent call last):
  File "/raid/projects/weustis/conda/envs/weustis/lib/python3.9/logging/__init__.py", line 227, in _releaseLock
    def _releaseLock():
KeyboardInterrupt:
  0%|                                                                                                                                        | 0/4039 [00:06<?, ?it/s]
Traceback (most recent call last):
  File "/raid/projects/weustis/conda/envs/weustis/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 990, in _try_get_data
    data = self._data_queue.get(timeout=timeout)
  File "/raid/projects/weustis/conda/envs/weustis/lib/python3.9/queue.py", line 179, in get
    raise Empty
_queue.Empty
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "/raid/projects/weustis/CrossConv/train.py", line 49, in <module>
    for x,y in tqdm(train_dataloader):
  File "/raid/projects/weustis/conda/envs/weustis/lib/python3.9/site-packages/tqdm/std.py", line 1180, in __iter__
    for obj in iterable:
  File "/raid/projects/weustis/conda/envs/weustis/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 521, in __next__
    data = self._next_data()
  File "/raid/projects/weustis/conda/envs/weustis/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 1186, in _next_data
    idx, data = self._get_data()
  File "/raid/projects/weustis/conda/envs/weustis/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 1142, in _get_data
    success, data = self._try_get_data()
  File "/raid/projects/weustis/conda/envs/weustis/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 1003, in _try_get_data
    raise RuntimeError('DataLoader worker (pid(s) {}) exited unexpectedly'.format(pids_str)) from e
RuntimeError: DataLoader worker (pid(s) 6112, 6113, 6115, 6116, 6123, 6124, 6125, 6126, 6127, 6128, 6130, 6131) exited unexpectedly